adapter/cmd/adapter/conf/config.toml

```diff
@@ -0,0 +1,92 @@
+[adapter] 
```

Comment:
Shall we keep only what we require for now here and remove others? just remove from the config.toml would be sufficient for the moment, we need to remove unnecessary configs from the code(default_config.go, etc) gradually.